draft-ietf-tls-esni.md

```diff
@@ -498,6 +553,8 @@ pad all other handshake messages that have sensitive-length fields. For example,
 if a client proposes ALPN values in ClientHelloInner, the server-selected value
 will be returned in an EncryptedExtension, so that handshake message also needs
 to be padded using TLS record layer padding.
+[[TODO: I'm thrown by "will also need". This seems like a required behavior
```

Comment:
Again, this is a RECOMMENDED policy. Clients could alternatively pad their inner CH to 1024B, or something. Padding is needed to hide size metadata. It doesn't affect the protocol mechanics.

Reply:
Resolving for now.